schema/deploy/appschema.sql

```diff
@@ -0,0 +1,7 @@
+-- Deploy todo_app:appschema to pg
+
+BEGIN;
+
+CREATE SCHEMA todo_app;
+
+COMMIT;
```

Comment:
We like our SQL keywords to be lowercase. Syntax highlighting is a thing nowadays, so we don't need to use casing to highlight keywords.